frontend/packages/dev-console/integration-tests/features/Knative/knative-Create_workload.feature

```diff
@@ -3,94 +3,96 @@ Feature: Create a workload of 'Knative Service' type resource
 
 Background:
    Given open shift cluster is installed with Serverless operator
-   And user is on dev perspective +Add page
-   And create the project "AUT-create-knative-worload"
+   And user is at developer perspecitve
+   And user has selected namespace "aut-create-knative-workload"
 
 
 @regression, @smoke
 Scenario: Knative resource type in git import add flow : Kn-01-TC03
-   Given user is on Add flow page
-   When user clicks on "From git" card
-   Then user redirects to page with header name "Import from git"
+   Given user is at Add page
+   When user clicks on From git card
+   Then user will be redirected to page with header name "Import from git"
    And Knaive Service option is displayed in Resources section
 
 
 @regression
 Scenario: Knative resource type in container image add flow : Kn-01-TC04
-   Given user is on Add flow page
-   When user clicks on "Container Image" card
-   Then user redirects to page with header name "Deploy Image"
+   Given user is at Add page
+   When user clicks on Container Image card
+   Then user will be redirected to page with header name "Deploy Image"
    And Knaive Service option is displayed in Resources section
 
 
 @regression
 Scenario: Knative resource type in docke file add flow : Kn-01-TC05
-   Given user is on Add flow page
-   When user clicks on "From Dockerfile" card
-   Then user redirects to page with header name "Import from Dockerfile"
+   Given user is at Add page
+   When user clicks on From Dockerfile card
+   Then user will be redirected to page with header name "Import from Dockerfile"
    And Knaive Service option is displayed in Resources section
 
 
 @regression
 Scenario: Knative resource type in catalog add flow : Kn-01-TC06
-   Given user is on Add flow page
-   When user clicks on "From Catalog" card
+   Given user is at Add page
+   When user clicks on From Catalog card
    And create the application with s2i builder image 
-   Then user redirects to page with header name "Create Source-to-Image Application"
+   Then user will be redirected to page with header name "Create Source-to-Image Application"
    And Knaive Service option is displayed in Resources section
 
 
 @regression, @smoke
-Scenario Outline: Create a work load from From Git card on Add page : Kn-02-TC01
-   Given user is on "<form_name>" form with header name "<header_name>"
-   When user type "<git_url>" into the "Git Repo url" text box
-   And select "Kantive Service" radio button on Add page
-   And click "Create" button on Add page
-   Then user redirects to topology page
-   And created workload is present in List View of topology page
+Scenario Outline: Create knative work load from From Git card on Add page : Kn-02-TC01
```

Comment:
```suggestion
Scenario: Create knative work load from From Git card on Add page: Kn-02-TC01
```

Reply:
Examples are used, Scenario outline is mandate